veros/core/friction.py

```diff
@@ -252,11 +252,24 @@ def harmonic_friction(vs):
         fxa = vs.cosu**vs.hor_friction_cosPower
         vs.flux_north[:, :-1] = vs.A_h * fxa[np.newaxis, :-1, np.newaxis] * (vs.u[:, 1:, :, vs.tau] - vs.u[:, :-1, :, vs.tau]) \
             / vs.dyu[np.newaxis, :-1, np.newaxis] * vs.maskU[:, 1:] * vs.maskU[:, :-1] * vs.cosu[np.newaxis, :-1, np.newaxis]
+        if vs.enable_noslip_latteral:
```

Comment:
The correct spelling is `lateral`. Please adjust.

Reply:
Sure :)